castai/resource_node_template_test.go

```diff
@@ -414,11 +414,11 @@ func TestAccResourceNodeTemplate_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "custom_instances_enabled", "false"),
 					resource.TestCheckResourceAttr(resourceName, "custom_label.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "custom_labels.%", "2"),
-					resource.TestCheckResourceAttr(resourceName, "custom_labels.custom-key-1", "custom-value-1"),
-					resource.TestCheckResourceAttr(resourceName, "custom_labels.custom-key-2", "custom-value-2"),
+					resource.TestCheckResourceAttr(resourceName, fmt.Sprintf("custom_labels.%s-1", rName), fmt.Sprintf("%s-1", rName)),
```

Comment:
by using "resourceName-1" for everything you're reducing test strength because now implementation can be seriously messed up and test might pass because every field expects same value.

Reply:
You are right, I thought about it as well. But my thinking was that if labels/taints get mixed up. It should be caught by different test.

I have added fixup with different values for each attribute.